scout/server/blueprints/cases/controllers.py

```diff
@@ -457,13 +462,13 @@ def gene_variants(store, variants_query, page=1, per_page=50):
 
     my_institutes = list(inst['_id'] for inst in user_institutes(store, current_user))
 
-    log.debug("Institutes allowed: {}.".format(my_institutes))
+    LOG.debug("Institutes allowed: {}.".format(my_institutes))
```

Comment:
Do we actually need these LOG.debugs now? I think they are leftovers? Perhaps just drop them or make log entries at warning level?

Reply:
I'll drop this and make a warning of those that actually are warnings

Reply:
Thanks - I'm not sure they were yours originally.